packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -241,6 +244,20 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
           this.distanceFromTop - container.scrollTop >= this._getStickyDistanceFromTopForFooter(container, footerStickyContainer);
       }
 
+      if (this.nonStickyContent) {
+        const isCurrentStateSticky = this.state.isStickyBottom || this.state.isStickyTop;
+        const isNextStateSticky = isStickyBottom || isStickyTop;
+        const nonStickyContentScrollWidth = this.nonStickyContent.scrollWidth;
+
+        if (!isCurrentStateSticky && isNextStateSticky) {
+          // calculating nonStickyPlaceHolder scrollWidth when current state is non-sticky & next state is sticky
+          this._nonStickyPlaceHolderScrollWidth = Math.min(nonStickyContentScrollWidth, this._nonStickyPlaceHolderScrollWidth);
+        } else if (isCurrentStateSticky && !isNextStateSticky) {
+          // calculating nonStickyPlaceHolder scrollWidth when current state is sticky & next state is non-sticky
+          this._nonStickyPlaceHolderScrollWidth = Math.min(nonStickyContentScrollWidth, this._nonStickyPlaceHolderScrollWidth);
+        }
```

Comment:
this can be simplified to if(isCurrentStateSticky != isNextStateSticky){
this._nonStickyPlaceHolderScrollWidth = Math.min(nonStickyContentScrollWidth, this._nonStickyPlaceHolderScrollWidth);
} for better understanding I have put 2 conditions